public/media-hunt/index.js

```diff
@@ -1 +1,3 @@
-console.log("Media hunt");
\ No newline at end of file
+const app = require("./mediahunt.js");
+
+app.run();
```

Comment:
Please remove this. This file will be responsible for parsing node commands.

Reply:
removed